### PR TITLE
Add celestia-gaia-stardb

### PR DESCRIPTION
**Pull request by SevenSpheres**
Adding code that generates files in CelestiaContent, at @LukeCEL's suggestion

### CODE REVIEW

**Comment by LukeCEL** [comment]
@375gnu This makes src/tools/stardb/buildstardb.pl and src/tools/xindex obsolete. Better to remove those files or to keep them?

**Comment by 375gnu** [comment]
I'd prefer to keep them for a while.

File: src/tools/celestia-gaia-stardb/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.vscode
```
Comment (375gnu) [inline src/tools/celestia-gaia-stardb/.gitignore:1]: this file is unneeded

Comment (SevenSpheres) [reply src/tools/celestia-gaia-stardb/.gitignore]: removed

**Comment by SevenSpheres** [comment]
This should probably be merged now

**Pull request merged**